features/itunes/api/getSongs.tsx

```diff
@@ -5,6 +5,8 @@ export interface SongItem {
   trackName: string;
   trackId: number;
   artistId: number;
+  trackTimeMillis: number;
+  artworkUrl100: string;
```

Comment:
Please check this style guide

https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms